packages/swap/src/routes/v2/utils.ts

```diff
@@ -127,19 +127,26 @@ export const getEgressStatusFields = async (
 ) => {
   const ignoredEgress = ignoredEgresses?.find((e) => e.type === type);
   const failureState = await getEgressFailureState(ignoredEgress, broadcast, type);
+  if (!egress && !broadcast && !failureState && !ignoredEgress) return null;
   return {
     ...(egress && {
       amount: egress.amount?.toFixed(),
       scheduledAt: egress.scheduledAt?.valueOf(),
       scheduledBlockIndex: egress.scheduledBlockIndex ?? undefined,
     }),
     ...(broadcast && {
-      sentAt: broadcast?.succeededAt?.valueOf(),
-      sentAtBlockIndex: broadcast?.succeededBlockIndex ?? undefined,
+      confirmedAt: broadcast?.succeededAt?.valueOf(),
+      confirmedBlockIndex: broadcast?.succeededBlockIndex ?? undefined,
       sentTxRef: broadcast?.transactionRef ?? egressTrackerTxRef,
```

Comment:
could just call this `txRef` now that we renamed `sent`

Reply:
done